packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -219,6 +220,17 @@ $ms-Persona-presenceBorder: 2px;
   }
 }
 
+.root.coinSizeWithPresence {
+  .presence {
+    height: $ms-Persona-presenceSizeFlex;
+    width: $ms-Persona-presenceSizeFlex;
+
+    .presenceIcon {
+      display: none;
```

Comment:
What if the custom size is big so there is actually room for the presenceicon?

Reply:
There wasn't a quick fix to get the icon to display at the correct scale for any size so the intention was to hide it for now (it was super tiny and in the wrong location). I talked to some folks in the office just now and it sounds like the custom coin size should be a large conversation anyway, especially if we need to go further than the quick fix route. I'll pull this out into a separate PR.